src/build/env.py

```diff
@@ -68,6 +72,19 @@ def _should_use_virtualenv() -> bool:
     )
 
 
+def _subprocess(*args: Any, **kwargs: Any) -> None:
+    """Invoke subprocess and output stdout and stderr if it fails."""
+    try:
+        subprocess.check_call(
+            *args,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            **kwargs,
+        )  # type: ignore  # https://github.com/python/mypy/issues/6799
+    except Exception as e:  # pragma: no cover
+        raise e
```

Comment:
This catch seems to do nothing :thinking:

Reply:
Don't you just want `subprocess.check_output` here?

Reply:
Yes, sorry. It accidentally got removed.

Reply:
You're catching a generic `Exception`, that's why it's not type hinted. Catch `CalledProcessError` and let everything else through.

Reply:
Yes, of course, sorry. I just woke up :sweat_smile: